metapackage_dependencies.rosinstall

```diff
@@ -4,3 +4,7 @@
 
 # When this file starts getting used, add it to the options (TARGET_WORKSPACE: "$TARGET_REPO_PATH metapackage_dependencies.rosinstall") of the following CI stages: arm64_build , amd64_build, clang-tidy, catkin_lint
 
+- git:
+    uri: https://github.com/uwrobotics/uwrt_mars_rover_hw_bridge.git
```

Comment:
i think u should use relative path here. makes it easier for people who use ssh (imo everyone should cuz of 2fa)

Reply:
are u sure URIs support relative paths? https://stackoverflow.com/questions/47307578/how-to-shallow-clone-a-local-git-repository-with-a-relative-path

Reply:
that link is talking specifically about file uris

Reply:
I've used this in a couple different repos before so i don't see why it wouldn't work

Reply:
it might be something to do with rosinstall?